index.html

```diff
@@ -10,97 +10,133 @@
 
 </head>
 <body>
-
+      
        <header class="header-bar">
 
         <nav class="nav-bar"><h3 class="Left-Logo">My Logo</h3>
-          <a><img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"></a></nav>
-       
+          <a><img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"></a>
+          <ul class="nav-list">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+           </ul>
+        </nav>
+
         </header> 
          
        <section class="headline-1">
 
+        <div class="background-image">
+
+         <div class="txt-arrange">
+
+           
            <h1 class="glad-txt">I'm Frederick Glad to see you!</h1>
-       
-          <p class="paragraph-txt">
-            I am a beginner student in the Microverse Program, this is my first week!
-            I'm learning right now how to implement flexbox properties in CSS. 
-            This is my first project using flexbox and setting up a mobile version skeleton
-          </p>
-
-          <div class="connect">
-            <h3 class="small-txt">LET'S CONNECT</h3>
-       </div>
-       <div class="media-Images">
-          <ul>
-            <li><a href="#"> <img src="Images/Twitter-icon.svg"   alt="Twitter Icon"></a></li>
-            <li><a href="#"> <img src="Images/Linkedin-icon.svg"   alt="Linkedin Icon"></a></li>
-            <li><a href="#"> <img src="Images/Elegant-M-Icon.svg"   alt="Micro Icon"></a></li>
-            <li><a href="#"> <img src="Images/GitHub-Desktop-icon.svg" alt="Github Icon"></a></li>
-            <li><a href="#"> <img src="Images/Peace-Emoji-Icon.svg"  alt="Peace Icon"></a></li>
-          </ul>    
-           </div>
 
-    </section>
+           <h1 class="title-replace">I'm Frederick<br>Glad to see you!</h1>
+           
+           <p class="paragraph-txt">
+             I am a beginner student in the Microverse Program, this is my first week!
+             I'm learning right now how to implement flexbox properties in CSS. 
+             This is my first project using flexbox and setting up a mobile version skeleton
+            </p>
+            
+            <div class="connect">
+              <h3 class="small-txt">LET'S CONNECT</h3>
```

Comment:
[OPTIONAL] Kindly wrap the `LET'S CONNECT` in an `a` tag also and link it to the contact form 🤗 so that a user can jump to the contact form anytime.